ioctl/newcmd/config/config_test.go

```diff
@@ -31,3 +32,34 @@ func TestInitConfig(t *testing.T) {
 	require.Equal(_supportedLanguage[0], cfg.Language)
 	require.Equal(filepath.Join(testPath, _defaultConfigFileName), cfgFilePath)
 }
+
+func TestConfigReset(t *testing.T) {
+	require := require.New(t)
+
+	info := newInfo(config.Config{
+		Wallet:           "wallet",
+		Endpoint:         "testEndpoint",
+		SecureConnect:    false,
+		DefaultAccount:   config.Context{AddressOrAlias: ""},
+		Explorer:         "explorer",
+		Language:         "RandomLanguage",
+		AnalyserEndpoint: "testAnalyser",
+	}, _defaultConfigFileName)
+
+	require.NoError(info.reset())
+
+	config.DefaultConfigFile = _defaultConfigFileName
+	cfg, err := config.LoadConfig()
```

Comment:
Don't use global because it may modify the true config data.
use info.loadConfig() and check data with set data lin40~46.  I think this can put into TestLoadConfig. it's just unit test.